doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,255 @@
+    width=500,
+    binary_string=True,
+    labels={'animation_frame': 'time point'},
+    title='Sample of in-vivo human cornea'
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Average over time
+# =================
+# First, we want to detect those dark spots where the data are lost. In
+# technical terms, we want to *segment* the dark spots (for
+# all frames in the sequence). Unlike the actual data (signal), the dark spots
+# do not move from one frame to the next; they are still. Therefore, we begin
+# by computing the time average of the image sequence. We shall use this
+# time-averaged image to segment the dark spots, the latter then standing out
+# with respect to the background (blurred signal).
+
+image_avg = np.mean(image_seq, axis=0)
+
+print(f'shape: {image_avg.shape}')
+
+fig, ax = plt.subplots(figsize=(6, 6))
+
+ax.imshow(image_avg, cmap='gray')
+ax.set_title('Time-averaged image')
+
+fig.tight_layout()
+
+#####################################################################
+# Use local thresholding
+# ======================
+# To segment the dark spots, we use thresholding. The images we are working
+# with have uneven illumination, which causes variations in the (absolute)
+# intensities of the foreground and the background, from one region to another
+# (distant) one. It is therefore more fitting to compute different threshold
+# values across the image, one for each region. This is called adaptive (or
+# local) thresholding, as opposed to the usual thresholding procedure which
+# employs a single (global) threshold for all pixels in the image.
+#
+# When calling the ``threshold_local`` function from the ``filters`` module,
+# we may change the default neighborhood size (``block_size``), i.e., the
+# typical size (number of pixels) over which illumination varies,
+# as well as the ``offset`` (shifting the neighborhood's weighted mean).
+# Let us try two different values for ``block_size``:
+
+thresh_1 = filters.threshold_local(image_avg, block_size=21, offset=15)
+thresh_2 = filters.threshold_local(image_avg, block_size=43, offset=15)
+
+mask_1 = image_avg < thresh_1
+mask_2 = image_avg < thresh_2
+
+#####################################################################
+# Let us define a convenience function to display two plots side by side, so
+# it is easier for us to compare them:
+
+def plot_comparison(plot1, plot2, title1, title2):
+    fig, (ax1, ax2) = plt.subplots(
+        ncols=2,
+        figsize=(12, 6),
+        sharex=True,
+        sharey=True
+    )
+    ax1.imshow(plot1, cmap='gray')
+    ax1.set_title(title1)
+    ax2.imshow(plot2, cmap='gray')
+    ax2.set_title(title2)
+
+plot_comparison(mask_1, mask_2, "block_size = 21", "block_size = 43")
+
+#####################################################################
+# The "dark spots" appear to be more distinct in the second mask, i.e., the
+# one resulting from using the larger ``block_size`` value.
```

Comment:
```suggestion
# The "dust spots" appear to be more distinct (brighter) in the second
# mask, i.e., the one created using the larger ``block_size`` value.
```

Reply:
Hm, they are not 'brighter' (see [above](https://github.com/scikit-image/scikit-image/pull/6853/files#r1228056336)). They (i.e., the selected regions, i.e., what the mask filters *in*) are more visible because they are *larger*.